services/node/testUtils/test-utils.js

```diff
@@ -0,0 +1,184 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const moment = require('moment')
+
+const dateFormat = 'YYYY-MM-DD'
+
+const templates = {
+  packageJsonVersionsTemplate: fs.readFileSync(
+    path.join(__dirname, `packageJsonVersionsTemplate.json`),
+    'utf-8'
+  ),
+  packageJsonTemplate: fs.readFileSync(
+    path.join(__dirname, `packageJsonTemplate.json`),
+    'utf-8'
+  ),
+}
+
+const getTemplate = template => JSON.parse(templates[template])
+
+const mockPackageData = (
+  packageName,
+  engines,
+  scope,
+  tag,
+  registry
+) => nock => {
```

Comment:
Let's destructure here given the quantity of params so that we don't have to rely on position and passing a lot of falsy values unnecessarily when there's a need to provide a value for `tag` or `registry`
```suggestion
const mockPackageData = ({
  packageName,
  engines,
  scope,
  tag,
  registry,
}) => nock => {
```

Reply:
yes my bad, will fix it